File: dart/resistor-color-duo/lib/resistor_color_duo.dart
class ResistorColorDuo {
  List<String> colors = [
    'black',
    'brown',
    'red',
    'orange',
    'yellow',
    'green',
    'blue',
    'violet',
    'grey',
    'white'
  ];

  int colorCode(String color) => colors.indexOf(color);

  int value(List<String> inputColors) {
    int firstIndex = colorCode(inputColors[0]);
    int secondIndex = colorCode(inputColors[1]);
    return int.parse('$firstIndex$secondIndex');
  }
}

File: python/resistor-color-duo/resistor_color_duo.py
_colors = [
            "black",
            "brown",
            "red",
            "orange",
            "yellow",
            "green",
            "blue",
            "violet",
            "grey",
            "white",
        ]


def value(colors):
    return int(str(color_index(colors[0])) + str(color_index(colors[1])))


def color_index(color):
    return _colors.index(color)
